courses/course-planning/degree-requirements.md

```diff
@@ -0,0 +1,42 @@
+---
+id: degree-requirements
+title: Degree Requirements
+sidebar_label: Degree Requirements
+slug: /course-planning/degree-requirements
+---
+
+# Degree Requirements
+
+The University offers three "main" degree types: The standard Bachelor of Computer Science, as well as the Applied Computing specialization, the Software Engineering specialization, and the Computer Information Systems specialization. All four-year degrees have 40 courses.
+
+The Appplied Computing specialization's main difference is that it removes many of the theoretical computer science requirements, as well as math, and replaces them with more "applied" courses and electives. If you know you aren't interested in those theoretical CS courses (e.x. they are often mandatory when applying for a Master's degree), this may be a good option to choose. Generally speaking, your employment prospects will be the same regardless of option.
```

Comment:
```suggestion
The Applied Computing specialization's main difference is that it removes many of the theoretical computer science requirements, as well as math, and replaces them with more "applied" courses and electives. If you know you aren't interested in those theoretical CS courses (e.x. they are often mandatory when applying for a Master's degree), this may be a good option to choose. Generally speaking, your employment prospects will be the same regardless of option.
```